docs/components/README.md

```diff
@@ -40,5 +40,7 @@ example: [Header (via settings)](./header.md#wagtail-site-settings)
 - [Expander & Expander Group](./expander.md)
 - [Header](./header.md)
 - [Inset Text](./inset_text.md)
+- [Captionable Images](./image.md)
```

Comment:
Two entries in the components list but only one component.
Which is it?
Decide if it should be called "Captionable Image" or "Image" and make sure that naming convention is consistent everywhere. (block name, template names, streamfield definition etc)